HookCatcher/views.py

```diff
@@ -73,32 +30,63 @@ def stateRepresentation(stateObj):
         'desc': stateObj.state_desc,
         'gitType': stateObj.git_source_type,
         'gitName': stateObj.git_source_name,
-        'gitCommitURL': GITHUB_PR_URL + stateObj.git_commit,
+        'gitCommitSHA': stateObj.git_commit[:7],
         'imgsOfState': stateObj.image_set.all()
     }
 
 
+# only take the information needed from JSON response
+def gitRepresentation(gitJSON):
```

Comment:
noted in another comment, but `gitJSON` is an object, not a JSON string. So you might name this `git_info` or similar...